src/ddl/covidcast.sql

```diff
@@ -138,4 +138,13 @@ CREATE TABLE `covidcast_meta_cache` (
   `epidata` LONGTEXT NOT NULL,
   PRIMARY KEY (`timestamp`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-INSERT INTO covidcast_meta_cache VALUES (0, '');
+INSERT INTO covidcast_meta_cache VALUES (0, '[]');
```

Comment:
Why do you need this line?

Reply:
empty string makes the JSON decoder puke.  an empty list is the "correct" cache for a new db w/o any data.